.pre-commit-config.yaml

```diff
@@ -46,4 +46,20 @@ repos:
 -   repo: https://github.com/pre-commit/mirrors-pylint
     rev: v2.3.1
     hooks:
-    -   id: pylint
+    - id: pylint
+      args:
+      - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+      - --output-format=colorized
+      - |-
+        --init-hook=
+        import os, platform, sys;
+        sys.path.insert(
+            0,
+            "".join([os.getenv("HOME"), "/virtualenv/python", platform.python_version(),
```

Comment:
Please don't use this hack and use a local mode as asked in the original task https://github.com/kpi-web-guild/django-girls-blog-schenkh/issues/7

Reply:
I was pretty sure I did that, let me check..